spotify_connect_scrobbler/credentials.py

```diff
@@ -58,52 +51,32 @@ def update(self, new_credentials):
 
 
 class Credentials:
+    """Main object for LastFM and Spotify credentials"""
 
     def __init__(self, lastfm, spotify, document_id=None):
         self.lastfm = lastfm
         self.spotify = spotify
         self.document_id = document_id
 
-    def save(self):
-        """Save credentials to MongoDB database.
-        """
+    def load_from_document(document):
+        lastfm = LastfmCredentials(
+            document['lastfm']['session_key'])
+        spotify = SpotifyCredentials(
+            document['spotify']['access_token'],
+            document['spotify']['token_type'],
+            document['spotify']['refresh_token'],
+            document['spotify']['scope']
+        )
+
+        return Credentials(lastfm, spotify, str(document['_id']))
+
+    def todict(self):
```

Comment:
Nah, I decided to rather change the other method to be `load_from_dict(credentials_dict)`.